Dockerfile

```diff
@@ -0,0 +1,23 @@
+# syntax=docker/dockerfile:1.0-experimental
+
+FROM node:12.12
+
+ENV PORT=3000
+ENV NODE_ENV=production
+
+WORKDIR /telegram-bot
+
+RUN wget https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh
```

Comment:
Why do we need this? I think it is too overkill just for waiting for redis

Reply:
it's just for waiting for Redis, how is it overkill?
We need a way to know if Redis is accepting connections when running the container. Otherwise it will crash